agenta-backend/agenta_backend/routers/evaluation_router.py

```diff
@@ -229,6 +231,61 @@ async def update_evaluation_scenario_router(
         raise HTTPException(status_code=500, detail=str(e)) from e
 
 
+@router.post(
+    "/evaluation_scenario/{evaluation_scenario_id}/ai_critique",
+    response_model=str,
+)
+async def evaluate_ai_critique(
```

Comment:
I don't think this is the right design.

The evaluate_ai_critique endpoint responsability should only be running the evaluation. It should not depend on the DB.
It's a simple endpoint that takes the correct_answer, inputs, outputs, LLM_template and return a score. IT does not need to know anything about the DB.